curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17d3bf86c76b9248c6eb4.md

```diff
@@ -31,7 +31,7 @@ Your `p` element's text should be `Click here to view more cat photos.` You have
 const extraSpacesRemoved = document
   .querySelector('p')
   .innerText.replace(/\s+/g, ' ');
-assert(extraSpacesRemoved.match(/click here to view more cat photos\.?$/i));
+assert(extraSpacesRemoved.match(/click here to view more cat photos\.$/i));
```

Comment:
```suggestion
assert(extraSpacesRemoved.match(/click here to view more cat photos\.?$/i));
```

Reply:
@Sboonny Exactly!  That is why I made the test more lenient originally.  I think most of the tests should be lenient when they can be in the first half of this project.  We are not testing whether or not someone can type out a sentence.  We are most concerned the user learns how to put text in a `p` element (in this case).  We do not want beginners to get hung up on a missing punctuation mark this earlier in the curriculum.

Reply:
@Sboonny @RandellDawson Thank you for your comments! I applied the suggestion.
Personally, I think it's rather confusing if what the hint says and what the test does are different, and it's a good opportunity to learn these 1 letter difference can matter in coding. But I take your suggestions.